### DISCUSSION TITLE
Get not full qualified element names from execute_mdx

### DISCUSSION THREAD

**Discussion by TM1si** [cubewise-code/tm1py#1019] [Q&A] [answered]
When requesting data from a cube

`data = tm1.cubes.cells.execute_mdx(mdx)
`
I get full qualified element names
with elements like 
`[tm1distr_step].[tm1distr_step].[1]
`
I can remove that
[tm1distr_step].[tm1distr_step].[ 

with a string operations, but this seems to be a lot of work for a common task.
Is there an build in function or an option to get only the name?

**Comment by MariusWirtz** [comment, marked as answer]
Yes. Just pass `element_unique_names=False` with the `execute_mdx` call. To simplify the value side of the dictionary you can pass `skip_cell_properties=True`

You can read about all arguments to the `execute_mdx` function here:
https://tm1py.readthedocs.io/en/latest/api.html#tm1-services
Just search through the document with `CTRL+F` for `execute_mdx`
